test/webdriverio/test/move_test.ts

```diff
@@ -145,6 +148,62 @@ suite('Move tests', function () {
       await this.browser.keys(Key.Escape);
     }
   });
+
+  // When a top-level block with no previous, next or output
+  // connections is subject to a constrained move, it should not move.
+  //
+  // This includes a regression test for issue #446 (fixed in PR #599)
+  // where, due to an implementation error in Mover, constrained
+  // movement following unconstrained movement it would result in the
+  // block unexpectedly moving (unless workspace scale was === 1)
```

Comment:
```suggestion
  // block unexpectedly moving (unless workspace scale was === 1).
```

Super nit on punctuation for consistency.